src/DirectionsMenu/SdkModel/DirectionsMenuModule.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright eeGeo Ltd (2012-2015), All Rights Reserved
+
+#include "DirectionsMenuModule.h"
+
+#include "IReactionControllerModel.h"
+#include "MenuModel.h"
+#include "MenuViewModel.h"
+#include "MenuSectionViewModel.h"
+
+namespace ExampleApp
+{
+    namespace DirectionsMenu
+    {
+        namespace SdkModel
+        {
+            DirectionsMenuModule::DirectionsMenuModule(Eegeo::Helpers::IIdentityProvider& identityProvider,
+                                               ExampleApp::Reaction::View::IReactionControllerModel& reactionControllerModel,
+                                             ExampleAppMessaging::TMessageBus& messageBus,
+                                               Metrics::IMetricsService& metricsService)
```

Comment:
Unnecessary metricsService - probably worth discussing with Mike what events we want to track for this